pool/web/view/page/page_admin_experiments.ml

```diff
@@ -309,7 +335,15 @@ let edit
     html
 ;;
 
-let detail experiment session_count Pool_context.{ language; csrf; _ } =
+let detail
+  experiment
+  session_count
+  invitation_templates
+  session_reminder_templates
+  sys_languages
+  Pool_context.{ language; csrf; _ }
+  =
+  let experiment_path = experiment_path experiment in
```

Comment:
the only occurrence of the experiment_path function, implement directly here.

Isn't there a similar function in our code base?